getVertices.m

```diff
@@ -24,28 +24,8 @@
 % code
 % ---------------------------------------------------------------
 
-% Check the type of surface data stored in the OpenEP datastructure and
-% accordingly access the points
-if isa(userdata.surface.triRep, 'TriRep')
-    FV.vert = userdata.surface.triRep.X;
-    FV.faces = userdata.surface.triRep.Triangulation;
-elseif isa(userdata.surface.triRep, 'triangulation')
-    FV.vert = userdata.surface.triRep.Points;
-    FV.faces = userdata.surface.triRep.ConnectivityList;
-elseif isa(userdata.surface.triRep, 'struct')
-    if ~isfield(userdata.surface.triRep, 'X')
-        error('OPENEP:invalidData', 'OPENEP/GETMESH: invalid data. userdata.surface.triRep must be one of: TriRep, triangulation or struct with fields .X and .Triangulation');
-    end
-    if ~isfield(userdata.surface.triRep, 'Triangulation')
-        error('OPENEP:invalidData', 'OPENEP/GETMESH: invalid data. userdata.surface.triRep must be one of: TriRep, triangulation or struct with fields .X and .Triangulation');
-    end
-    FV.vert = userdata.surface.triRep.X;
-    FV.faces = userdata.surface.triRep.Triangulation;
-else
-    error('OPENEP:invalidData', 'OPENEP/GETMESH: userdata.surface.triRep must be one of: TriRep, triangulation or struct');
-end
-
-vertices = FV.vert;
-[~, isVertUsed] = repack(getMesh(userdata));
+tr = getMesh(userdata);
+vertices = tr.X;
```

Comment:
specify type parameter for getMesh at line 27 to be 'triangulation' ; and refer to the Points property of the resulting returned Triangulation object